src/desktop/package.json

```
@@ -196,7 +202,11 @@
     "moduleNameMapper": {
       "\\.(scss)$": "<rootDir>/__mocks__/styleMock.js",
       "\\.(svg)$": "<rootDir>/__mocks__/fileMock.js",
-      "\\.(md)$": "<rootDir>/__mocks__/markdownMock.js"
+      "\\.(md)$": "<rootDir>/__mocks__/markdownMock.js",
+      "^react": "<rootDir>/../shared/node_modules/react",
```

```suggestion
      "^react$": "<rootDir>/../shared/node_modules/react",
```

From [Jest docs](https://jestjs.io/docs/en/configuration#modulenamemapper-object-string-string):
>The order in which the mappings are defined matters. Patterns are checked one by one until one fits. The most specific rule should be listed first.
>
>Note: If you provide module name without boundaries `^$` it may cause hard to spot errors. E.g. `relay` will replace all modules which contain `relay` as a substring in its name: `relay`, `react-relay` and `graphql-relay` will all be pointed to your stub.

Because this rule is before others, anything with `react` in the name is pointed to `<rootDir>/../shared/node_modules/react` -- including `react-dom` (see https://regexr.com/4ia3p). This is causing all tests to fail because `enzyme-react-adapter-16` depends on `react-dom` but is pointed to `react`. Adding `$` fixes all tests.